main.py

```diff
@@ -0,0 +1,24 @@
+# main.py
+from animals import Animal, Cat, Dog, Zoo, ZooKeeper
+
+if __name__ == "__main__":
+    # create animal objects
+    cat = Cat("Whiskers", 5)
+    dog = Dog("Buddy", 3)
+    
+    # create animal age objects for print statement
+    cat.display_age()
+    dog.display_age()
+
+    # Create a Zoo and add animals
+    zoo = Zoo()
+    zoo.add_animal(cat)
+    zoo.add_animal(dog)

```

Comment:
technically, you don't need to utilize the add_animal function on the zoo directly, since it's not the zoo's responsibility to call this method, but the zookeepers. You'll still need the functionality, but you don't need to expose and use it in the Zoo class, but rather the ZooKeeper only

Reply:
Thank you! Removed add_animal functions.